### DISCUSSION TITLE
Testing needed for a branch

### DISCUSSION THREAD

**Discussion by KubaO** [wxMaxima-developers/wxmaxima#1452] [General]
I'd appreciate anyone willing to test https://github.com/wxMaxima-developers/wxmaxima/tree/feature/KubaO/reorder-inner-cells - this change should not affect line breaking of long expressions (I hope), so that's what would need to be tested. I'm factoring out smaller, less intrusive changes from the big nextToDraw pointer removal, to make it easier to catch the source of misbehavior.

**Comment by gunterkoenigsmann** [comment]
Seems to work fine, and additionally resolves the bug I was hunting. => Good work!

**Reply by KubaO** [reply]
Thank you! It's a bit troubling that it resolves a bug, since all I changed was to reorder internal cells to be in draw order, since that's necessary for the subsequent removal of nextToDraw pointer. This would seem to indicate that some code, somewhere, is depending on this order. Could you say a bit about what the bug is? I'd want to understand why it got fixed, and perhaps look for other code where similar assumptions could potentially lead to same bugs?

**Comment by gunterkoenigsmann** [comment]
Sometimes I had the impression that if I selected code in the output the selection was drawn with a small offset. That seems no more to be the case.

**Reply by KubaO** [reply]
Sounds good for me - this could be an unintended positive consequence, but doesn't signify some big problem. So all is good for now. We're on our way to getting rid of nextToDraw :) And from there, next step is Configuration split into stuff that actually is configuration (and will remain Configuration), and CellContext which will contain stuff like wxDC etc.
